components/browser/awesomebar/src/main/res/layout/mozac_browser_awesomebar_item_generic.xml

```diff
@@ -44,13 +45,12 @@
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="20dp"
-        android:layout_marginTop="8dp"
+        android:layout_marginBottom="4dp"
         android:layout_marginEnd="8dp"
         android:ellipsize="end"
         android:lines="1"
         android:textSize="12sp"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
Was this removed intentionally? 🤔

Reply:
oops, didn't see that when reviewing, just saw the margin :(. Here's a PR to undo just that line which looks unintentional to me too: https://github.com/mozilla-mobile/android-components/pull/3006 @bwinton can confirm :)

Reply:
Do! I think I meant to remove the line above, in a previous attempt to center things using a `chainPacked` constraint… So yeah, this should be put back. Thank you!